pyproject.toml

```diff
@@ -37,6 +38,7 @@ deps =
     dj22: django>=2.2,<2.3
     dj30: django>=3.0,<3.1
     dj31: django>=3.1,<3.2
+    dj32: django>=3.2,<4.0
```

Comment:
Although the next release will be 4.0 this should probably be defined as:

    dj32: django>=3.2,<4.0

This is more consistent and avoids problems if plans change or tools get funny release versions. Has happened before, will happen again. Maybe not with Django, but a lot of places had lax version checks back when Python 3 became mainstream as well as when Django 3 was released and both produced a bunch of issues in libraries.